### DISCUSSION TITLE
Compose: Which operator is this?

### DISCUSSION THREAD

**Discussion by claus-01** [ImageMagick/ImageMagick#2290] [Help]
Hi,

I have been successful using the following code from the [IM > Usage page](https://www.imagemagick.org/Usage/) to optimize scans:

`$ convert [source] \( +clone -blur 0x20 \) \
-compose Divide_Src -composite  [result]`

What I would like to know is, which [Duff-Porter operator](https://www.imagemagick.org/Usage/compose/#duff-porter) is `Divide_Src`?

Claus

**Comment by snibgo** [comment]
It isn't a Duff-Porter method. It is what ImageMagick calls a "mathematical method". See the top of the documentation page you linked for the categories of compose methods.

**Reply by claus-01** [reply]
Oh, I see. Thanks ;-)
